README.md

```diff
@@ -108,15 +115,18 @@ This example demonstrates how Personal-Graph can be used to build long-term memo
 
 ### Creating and Querying a Knowledge Graph
 ```py
-from personal_graph import Graph, natural
+from personal_graph.graph import Graph
+
+your-litellm-client = LLMClient(client="llm-client", model_name="llm-model-name")
+your-embedding-client = EmbeddingClient(client="embedding-client", model_name="embedding-model-name", dimensions="model-dimension")
```

Comment:
use default values in all the examples except in the example showing how to use any model using litllm

Reply:
Done